woocommerce/woocommerce
# Prevent prefilling of the billing address with the shipping address in the Checkout block for the Guest Shopper

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds a new enhancement to the Checkout Block. It prevents the billing address from automatically filling in with the shipping address for the guest shopper.  

Closes #42140.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.  Open the site in a private/incognito browser window.
2. Add a product to the Cart.
3. Visit the Checkout Block page.
4. Fill out the shipping address.
5. Uncheck the Use same address for billing checkbox.
6. Confirm that the billing address form is empty.
7. Confirm State and Country is pre-filled. 




### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Prevent prefilling of the billing address with the shipping address in the Checkout block for the Guest Shopper. 

#### Comment 

</details>
